voter/management/commands/voter_process_snapshot.py

```diff
@@ -122,129 +126,139 @@ def reset_file(file_tracker):
     file_tracker.save()
 
 
-@transaction.atomic
+# @transaction.atomic
```

Comment:
Are you thinking we'll bring this back? Or should we just remove it?

Reply:
hmm no i don't want to bring it back. if anything i want to make the imports resumable, which we can't do if the whole thing is one transaction.

Reply:
commented line removed